pages/api/v1/activity-feed/subscribed-posts.ts

```
@@ -0,0 +1,335 @@
+					],
+					isStatus
+				);
+			} else {
+				proposalTimeline = getTimeline(group?.proposals, isStatus) || [];
+			}
+			const postDocRef = postsByTypeRef(network, ProposalType.REFERENDUM_V2 as ProposalType).doc(String(postId));
+
+			const commentsQuerySnapshot = await postDocRef.collection('comments').where('isDeleted', '==', false).count().get();
+
+			const postDoc = await postDocRef.get();
+			if (postDoc && postDoc.exists) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)